pkg/metrics/install.go

```diff
@@ -107,7 +106,7 @@ func getEndpoint(license *kotsv1beta1.License) string {
 		return license.Spec.Endpoint
```

Comment:
this doesn't feel right

Reply:
this isnt new code

Reply:
I know, but it feels like the env var should override this endpoint if present

Reply:
oh i see... yes